btr-web/btr-main-app/components/bcros/AddressDisplay.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div data-cy="address-display">
+    <div
+      v-for="addressLine, i in addressData"
```

Comment:
`v-for="addressLine, i in addressData"`
==> `v-for="(addressLine, i) in addressData"`
I would use brackets here, it shows better variable assignement